### PR TITLE
[IND-416] Add default fee for v4 python client

### PR DESCRIPTION
**Pull request by dydxwill**

### CODE REVIEW

File: v4-client-py/v4_client_py/chain/aerial/client/utils.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def prepare_and_broadcast_basic_transaction(
     gas_limit: Optional[int] = None,
     memo: Optional[str] = None,
     broadcast_mode: BroadcastMode = None,
+    fee: Optional[str] = "5000dv4tnt",
```
Comment (Christopher-Li) [inline v4-client-py/v4_client_py/chain/aerial/client/utils.py:21]: Kind of open questions, but shouldn't fee be required? I think on the protocol it is required. Additionally on mainnet will the token be called `dv4tnt`? Not sure if we want to default to this value

Comment (vincentwschau) [reply v4-client-py/v4_client_py/chain/aerial/client/utils.py:21]: Placing orders does not require a fee.

Comment (dydxwill) [reply v4-client-py/v4_client_py/chain/aerial/client/utils.py:21]: set the fee to be 0 if placing an order. We can change the default token when mainnet comes around?

Comment (vincentwschau) [reply v4-client-py/v4_client_py/chain/aerial/client/utils.py:21]: May make more sense to define a default `coin` based on the network in the constants [here](https://github.com/dydxprotocol/v4-clients/blob/main/v4-client-py/v4_client_py/clients/constants.py).

Comment (dydxwill) [reply v4-client-py/v4_client_py/chain/aerial/client/utils.py]: done

File: v4-client-py/v4_client_py/clients/constants.py [hunk 2]
```diff
@@ -96,6 +96,9 @@
 
 BECH32_PREFIX = 'dydx'
 
+# ------------ DEFAULT TOKEN ------------
```
Comment (vincentwschau) [inline v4-client-py/v4_client_py/clients/constants.py:99]: nit: Change the comment to indicate it's the testnet token.

File: v4-client-py/v4_client_py/clients/constants.py [hunk 3]
```diff
@@ -96,6 +96,9 @@
...
+DEFAULT_TOKEN = "dv4tnt"
```
Comment (vincentwschau) [inline v4-client-py/v4_client_py/clients/constants.py:100]:
```suggestion
TESTNET_TOKEN = "dv4tnt"
```

**Comment by vincentwschau** [comment]
Can we set the fee_denomination [here](https://github.com/dydxprotocol/v4-clients/blob/main/v4-client-py/v4_client_py/chain/aerial/config.py#L32) and use that network config in some way instead?

File: v4-client-py/v4_client_py/chain/aerial/config.py [hunk 4]
```diff
@@ -93,7 +108,7 @@ def fetchai_stable_testnet(cls):
 
         :return: fetchai stable testnet. For now dorado is fetchai stable testnet.
```
Comment (vincentwschau) [inline v4-client-py/v4_client_py/chain/aerial/config.py:109]: nit: Update this comment?

File: v4-client-py/v4_client_py/chain/aerial/client/utils.py [hunk 5]
```diff
@@ -28,6 +28,8 @@ def prepare_and_broadcast_basic_transaction(
     :param account: The account
     :param gas_limit: The gas limit
     :param memo: Transaction memo, defaults to None
+    :param broadcast_mode: Broadcast mode, defaults to None
+    :param fee: Transaction fee, defaults to 5000
```
Comment (vincentwschau) [inline v4-client-py/v4_client_py/chain/aerial/client/utils.py:32]: nit: Note that the denomination is from the network config.

**Review by vincentwschau — APPROVED** [review]

File: v4-client-py/v4_client_py/chain/aerial/config.py (hunk 4 above)
Comment (dydxwill) [reply v4-client-py/v4_client_py/chain/aerial/config.py:109]: done

File: v4-client-py/v4_client_py/chain/aerial/client/utils.py (hunk 5 above)
Comment (dydxwill) [reply v4-client-py/v4_client_py/chain/aerial/client/utils.py:32]: done

**Pull request merged**